### DISCUSSION TITLE
Grid missing 3 sensors

### DISCUSSION THREAD

**Discussion by influence2006** [Security-Onion-Solutions/securityonion#5913] [Unsupported Versions] [answered]
Good afternoon.

I am running SO 2.3.80 distributed setup.
My grid shows 9 sensors in an okay status
I have a total of 12 sensors and seem to be missing  3 of them. I do show them reporting and in Grafana.
I am just wondering what I can do to get the missing sensors into the grid view?

**Comment by palichx** [comment]
check detail status: 
$`salt "*name_problem_sensor*" cmd.run "so-status"`

**Reply by influence2006** [reply]
I do run 
```
salt \* so.status
```

and all sensors report back their status. 3 just don't seem to want to go into the grid.

**Comment by dougburks** [comment, marked as answer]
Have you checked the SOC logs in `/opt/so/log/soc/` for any additional clues?

Have you tried restarting SOC with `sudo so-soc-restart`?

Have you tried rebooting your manager?

**Reply by influence2006** [reply]
I do not see the IP for the sensors missing in the logs for sensoroni 
Sync has no new messages for today. 
I have restarted the manager/sensor with so-restarts and a reboot.

**Reply by dougburks** [reply]
Are there any network firewalls between the 3 missing sensors and your manager?  If so, do you have the necessary firewall rules to allow sensors to connect to the manager as shown at https://docs.securityonion.net/en/2.3/firewall.html#node-communication?

From one of the 3 missing sensors, can you connect to port 443 on the manager?

**Reply by influence2006** [reply]
Yes, I can do a curl with no issues (sorry no GUI) Log messages are flowing in from all 3 of the sensor to kibana and grafana.

**Reply by dougburks** [reply]
Are there any ways in which the 3 missing sensors differ from the other sensors? Hostname, configuration, network placement, etc.

**Reply by influence2006** [reply]
They actually exact clones of each other I started with one sensor and moved them across to different esxi hosts.

**Reply by dougburks** [reply]
At what point were they cloned? Could the fact that they were cloned mean that they could have some duplicate configuration (like hostname)?

**Reply by influence2006** [reply]
They were all cloned at the start of installation. Then I go in and finish installation. All sensors have their own hostname which matches kibana and grafana. The only area that isn't right is the grid area. All sensors report back with different data under the right host name.

**Reply by dougburks** [reply]
Have you checked `/opt/so/log/sensoroni/` on the problematic sensors?

**Reply by influence2006** [reply]
Seems like its a DNS issue changing it made all the difference. I don't get why this would rely on DNS :-/

**Reply by dougburks** [reply]
![image](https://user-images.githubusercontent.com/1659467/139488511-c8fcb97b-2f5b-4f82-8791-e6bfd8035521.png)

**Comment by influence2006** [comment]
Yes lol. After I checked the logs and see that somethings were pointing to
8.8.8.8... I started to just shake my head.
